alternator/ttl.cc

```diff
@@ -93,7 +93,7 @@ future<executor::request_return_type> executor::update_time_to_live(client_state
     if (v->GetStringLength() < 1 || v->GetStringLength() > 255) {
         co_return api_error::validation("The length of AttributeName must be between 1 and 255");
     }
-    sstring attribute_name(v->GetString(), v->GetStringLength());
+    sstring attribute_name = rjson::to_sstring(*v);
```

Comment:
This change is correct, but this line was also correct before this patch.

Reply:
yes, I tried to unify, because it's easier to maintain if code is similar